### PR TITLE
Reflect changes in lrtable's state_action function.

### PR DESCRIPTION
**Pull request by ltratt**
https://github.com/softdevteam/lrtable/pull/85 needs to be merged before this. The tests will probably need kicking after that other PR is merged before the tests on this PR succeed.

### CODE REVIEW

**Comment by snim2** [comment]
Kicking the build now...

File: src/lib/corchuelo.rs [hunk 1]
```diff
@@ -235,7 +235,7 @@ impl<'a, TokId: PrimInt + Unsigned> Recoverer<TokId> for Corchuelo<'a, TokId>
                 }
 
                 let la_tidx = parser.next_tidx(n.la_idx);
```
Comment (snim2) [inline src/lib/corchuelo.rs]: If `la_tidx` is not used elsewhere is the assignment useful?

File: src/lib/mf.rs [hunk 2]
```diff
@@ -226,7 +226,7 @@ impl<'a, TokId: PrimInt + Unsigned> Recoverer<TokId> for MF<'a, TokId>
                 }
 
                 let la_tidx = parser.next_tidx(n.la_idx);
```
Comment (snim2) [inline src/lib/mf.rs]: If `la_tidx` is not used elsewhere is the assignment useful?

File: src/lib/mf.rs [hunk 3]
```diff
@@ -370,7 +370,7 @@ impl<'a, TokId: PrimInt + Unsigned> MF<'a, TokId> {
         let la_tidx = self.parser.next_tidx(n.la_idx);
```
Comment (snim2) [inline src/lib/mf.rs:370]: Likewise.

**Comment by snim2** [comment]
Otherwise looks fine.

File: src/lib/mf.rs (hunk 3 above)
Comment (ltratt) [reply src/lib/mf.rs:370]: I agree with the other two; this one just makes the formatting almost impossible.

**Comment by ltratt** [comment]
See if that new commit works for you.

**Comment by snim2** [comment]
OK, time to squash!

**Review by snim2 — APPROVED** [review]

**Comment by ltratt** [comment]
Squashed!

**Pull request merged**